Properly configure C* and KairosDB services in CentOS 7

nacx opened the pull request:
Depends on a change to the C* cookbook. Once https://github.com/michaelklishin/cassandra-chef-cookbook/pull/353 is merged and released we'll need to remove the reference to the cookbook fork in the Berksfile.

Once there is a better package for KairosDB we should also remove the custom systemd unit file from the monitoring recipes.

Fixes #68.

chirauki commented on recipes/install_monitoring.rb:
```diff
@@ -35,6 +33,45 @@
+	[Unit]
+	Description=KairosDB is a fast distributed scalable time series database written on top of Cassandra.
+	Requires=cassandra.service
+	After=cassandra.service
```
C* and kairos could be on different hosts. This unit should not depend on cassandra?

chirauki commented on recipes/install_monitoring.rb:
```diff
@@ -35,6 +33,45 @@
+
+	[Service]
+	Type=forking
+	User=root
```
Let's create a user for Kairos and run the service non-root.

nacx commented:
I've rebased to the latest master and split the KairosDB resources to a separated recipe.

I'm only creating the user on CentOS 7 because in CentOS 6 we use the script as it comes from the package and I don't want to change and tune its permissions. In CentOS 7, we create the `kairosdb` group and user to run the service. We also fix the ownership and permissions of the KairosDB files.
Note that I've had to move the PID file to its own directory `/var/run/kairosdb` to be able to properly set the permissions; users don't have permissions to write files directly to `/var/run`.

nacx commented on test/integration/monitoring/serverspec/monitoring_services_spec.rb:
```diff
@@ -49,3 +51,9 @@
+
+describe 'Monitoring services for CentOS 7', if: os[:release].to_i >= 7 do
+  it 'has the kairosdb service running as the kairosdb user' do
+    expect(command('ps h -u kairosdb -o cmd').stdout).to match(/.*org\.kairosdb\.core\.Main.*/)
```
Quick note: ServerSpec does not provide a way to check which user runs a service and its `process` directive cannot be used because the process is `java`. Instead, we use this command to list the processes owned by the `kairosdb` user (printing only the command and no headers) and check that the command is actually the KairosDB one.
